ietf-layer0-types-ext.yang

```diff
@@ -390,6 +408,31 @@ module ietf-layer0-types-ext {
       resolution bandwidth";
   }
 
+  typedef fiber-type {
+    type enumeration {
+      enum G.652 {

```

Comment:
I am not sure if there is any convention in YANG modeling to use or not use "." in variable names. For example, in iana-if-type.yang it uses "identity ieee8023adLag" as opposed to "802.3". Maybe just removing ".", to make it like G652, G653.

Reply:
Section 4.3.1 of RFC8407 says:

>    Identifiers SHOULD follow a consistent naming pattern throughout the
>    module.  Only lowercase letters, numbers, and dashes SHOULD be used
>    in identifier names.  Uppercase characters, the period character, and
>    the underscore character MAY be used if the identifier represents a
>    well-known value that uses these characters.  YANG does not permit
>    any other characters in YANG identifiers.

I think that an ITU-T Recommendation number can be seen as "well-known value" but I agree we may need to have a consistent approach at least in the YANG models developed in CCAMP WG

Reply:
agreed

Reply:
maybe we can think to add a note in the next IETF presentation to ask WG their opinion and be sure consistency has to be checked

Reply:
Tracked as https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext/issues/26